cloudhub-2/modules/ROOT/pages/ch2-ha-dr.adoc

```diff
@@ -0,0 +1,269 @@
+
+*Recovery Point*
+
+* For integrating systems to resend event or data of in-flight transactions, capture recovery points in logs and monitoring.
+
+== Runtime Plane High Availability and Disaster Recovery Summary
+
+[%header%autowidth.spread]
+|===
+|API Type |Configuration
+|High-Impact Stateless API | Use an active-active setup with active replicas in two regions and an external load balancer.
+|High-Impact Stateful API a|
+* Use a hot standby setup with applications deployed and started in both regions but processing requests only in one.
+* Use an external load balancer to route traffic, and perform health checks to determine DR and switch traffic.
+|Medium-Impact Stateless and Stateful API a|
+* Use cold/warm standby, with applications deployed to both regions.
+* Stop or scale down the applications or replicas in the secondary region.
+|===
+
+== CloudHub 2.0 High Availability and Disaster Recovery Improvements
+
+CloudHub 2.0 Improvements over CloudHub include:
+
+* Region and control plane decoupling from the runtime plane.
+* Clustering allows you to configure HA while running the process on a single primary replica.
+* Horizontal Pod Autoscaling (HPA) for warm standby optimizes usage by elastic scaling if a failover is required, even if the control plane is down.
+* Active/Passive applications: Configure schedulers, listening connectors, such as MQ listener, DB listener, FTP listeners, and Salesforce listener, to be in a stopped or inactive state. 
++
+Externalize DR flags and expose REST endpoints to enable and disable flows via Groovy script as needed.
+
+== Disaster Recovery for Regional Services
+
+Certain MuleSoft services are regional. This impacts their DR strategy.
+
+=== Anypoint MQ
+
+This is a regional service that runs in the same region as the application. It provides out-of-the-box (OOTB) region failover to a fallback region for standard queues. If an application's RTO tolerates processing in-flight messages only after the primary site is backed up, MQ failover is sufficient. Otherwise, configure a backup application in an Active/Passive setup, and resend in-flight events and data to the failover queue during a traffic switch. During backup with VM Queues and Anypoint MQ, data can be lost for a few minutes, depending on the configuration.
+
+=== Object Store v2 (OSv2)
+
+This is a regional service running in the same region as the application. OSv2 doesn't provide any region failover functionality. An application deployed in a different region can't access it. Bring your own external storage to access or backup data from different regions. OSv2 doesn't retrieve data for regular backups. Caching in ObjectStore, used for token management between calls, is region-specific, and you can't transfer it to a secondary region.
+
+== Considerations for Disaster Recovery Implementations
+
+Implementing a Disaster Recovery (DR) strategy for MuleSoft applications requires careful consideration of several key factors.
```

Comment:
I think this should be "Mule applications" or "applications deployed to CloudHub 2.0".